### ISSUE TITLE
Where are .NET 6 dependencies resolved for csproj files?

### ISSUE THREAD

**Issue by Flohack74** [fsprojects/Paket#4114]
Hi,

we have some issues getting dependencies recognized by Visual Studio (while they are in the list, the compiler cannot find them, and we want to rule out framework incompatibilities)
So how does the csproj know about those deps:

When they are no longer directly in the .csproj file? How does the include work so that VIsual Studio can show them? How can I make sure that all references are really seen by msbuild etc.

UPDATE Now all references have disappeared and I cant bring them back:
![grafik](https://user-images.githubusercontent.com/16186470/150317534-bc493cc7-6696-4999-bf8d-210cb4007a0b.png)

**Comment by Flohack74** [comment]
Paket dependencies:
```
source https.//ourownsource.here
source https://api.nuget.org/v3/index.json

storage: none
framework: auto-detect
nuget Common.Logging
```
paket.references
```
Common.Logging
```

**Comment by jckjaer** [comment]
I am currently experiencing the same issue, although in F#.

I created a new F# console application with .NET 6.
I ran the following commands:
`dotnet new tool-manifest`
`dotnet tool install Paket`
`dotnet tool restore`
`dotnet paket init`
`dotnet paket add Dapper --interactive`
`dotnet paket restore`
`dotnet paket install`

In my project, I see no references and I cannot use them.
![image](https://user-images.githubusercontent.com/6909686/151126896-abb70a0e-3702-4909-9228-4e5ac463447b.png)

I am very new to using Paket, perhaps I am doing something wrong.

**Comment by jckjaer** [comment]
I have since gotten it to work by deleting my entire .nuget directory. After this, `dotnet paket install` worked perfectly.
My .nuget folder was located at `%UserProfile%\.nuget`.

**Comment by jckjaer** [comment]
For anyone else still not getting it to work:

I created a new .NET 6 project that again failed to see the dependencies added via paket.

I then looked in `paket.dependencies` and noticed that the "framework" was specified to .NET 5.
I changed it to `framework: auto-detect`. 
Then I deleted the following folders to ensure nothing was cached in Visual Studio:

- `paket-files`
- `packages`
- `.paket`
- `.vs`
- `bin`
- `obj`

Afterwards, doing the following commands worked: `dotnet paket restore` and `dotnet paket install`.
